packages/react-data-grid/src/index.ts

```diff
@@ -7,7 +7,7 @@ import * as formatters from './formatters';
 import * as editors from './common/editors';
 import * as shapes from './common/prop-shapes';
 import * as _utils from './common/utils';
-import * as _constants from './common/constants';
+import * as _constants from './common/enums';
```

Comment:
May be we should flatten the export so we can do the following 

```
import ReactDataGrid, { HeaderRowType, DropdownEditor } from `react-data-grid` 
```

Currently we are forced to use 

```
const { HeaderRowType } = _constants ;
```

This does not need to go in this PR but something to think about